<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1106</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="Line" name="line_14">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>10</y>
     <width>20</width>
     <height>581</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_18">
   <property name="geometry">
    <rect>
     <x>878</x>
     <y>100</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="date_change_format">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>270</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Change Format</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>891</x>
     <y>156</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Date :</string>
   </property>
  </widget>
  <widget class="QLabel" name="date_lb">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>155</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="date_date_le">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>200</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>2019-12-07</string>
   </property>
  </widget>
  <widget class="QPushButton" name="date_info">
   <property name="geometry">
    <rect>
     <x>1017</x>
     <y>154</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Display Info</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="Line" name="line_19">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>330</y>
     <width>221</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="date_edit_min">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>300</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit Min</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="date_edit">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>120</y>
     <width>201</width>
     <height>22</height>
    </rect>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="date_disp_format_le">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>240</y>
     <width>201</width>
     <height>20</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>yy-MMMM-dddd</string>
   </property>
  </widget>
  <widget class="QPushButton" name="date_print_range">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>300</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Print Range</string>
   </property>
  </widget>
  <widget class="QPushButton" name="date_edit_max">
   <property name="geometry">
    <rect>
     <x>1020</x>
     <y>270</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit Max</string>
   </property>
  </widget>
  <widget class="QPushButton" name="date_set_date">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>200</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Date</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
